Bottom: 9027c854ae1167ad6be502cde8ee1a6605cffd50
Top:    e80fe28049bac319e861bd2a59841879a1a7d11b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-07 01:46:24 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
